sql/core/src/main/scala/org/apache/spark/sql/execution/command/views.scala

```diff
@@ -116,7 +116,7 @@ case class CreateViewCommand(
     if (viewType == LocalTempView) {
       val aliasedPlan = aliasPlan(sparkSession, analyzedPlan)
       if (replace && needsToUncache(catalog.getRawTempView(name.table), aliasedPlan)) {
-        logInfo(s"Try to uncache ${name.quotedString} before replacing.")
+        logDebug(s"Try to uncache ${name.quotedString} before replacing.")
```

Comment:
I changed this to `logDebug` because the permanent view was using it: https://github.com/apache/spark/blob/9ec8696f117a5c43d24aae3188f07ddd91b69d2d/sql/core/src/main/scala/org/apache/spark/sql/execution/command/views.scala#L174

If `logInfo` is better, I can change these to `logInfo` instead.